app/move/index.js

```diff
@@ -13,6 +13,7 @@ const wizardConfig = {
   controller: Form,
   name: 'new-move',
   journeyName: 'new-move',
+  journeyPageTitle: 'Create a move',
```

Comment:
not sure how it works but it _looks_ like this needs to be localisable?

Reply:
Good spot. Glad you keep remembering this :)